zorkington-game.js

```diff
@@ -0,0 +1,503 @@
+    // first create takeable items
+    // room number, item name, item description, takeable boolean, hasFiniteLife boolean, affectsRoomLock boolean, affectsHealthBoolean, life
+    itemCollection.addItem(new Item(1, 'note', 'Welcome you to my home stranger.\nLet\'s see if you can find your way out!,', true, false, false, false));
+    itemCollection.addItem(new Item(1, 'flashlight', 'Your flashlight is on!\nYou can now see a small table next to a comfortable looking chair meant for reading.\nOn the table is a book with a faded "X" on the spine, underneath it looks to be a handwritten note.', true, true, false, false, 100));
+    itemCollection.addItem(new Item(1, 'book', 'You slide the book onto the shelf in the empty space between the "W" & "Y" encyclopedia books.\nYou hear the click of a lock being undone. The bookcase creaks open!\nYou are now able to move East to the next room!', true, false, true, false));
+    itemCollection.addItem(new Item(2, 'batteries', 'You put the fresh batteries in your flashlight.\nYour flashlight is at full power!', true, false, false, false));
+    itemCollection.addItem(new Item(3, 'screwdriver', 'You slide the narrow end of the screwdriver between the dining room doors and slide it up.\nThe latch on the other side lifts!\nYou\'ve unlocked the door to the dining room!', true, false, true, false));
+    //itemCollection.addItem(new Item(3, 'cookies', 'yummy cookies', true, false, false, true));
+    itemCollection.addItem(new Item(3, 'knife', '', true, false, false, true));
+    //itemCollection.addItem(new Item(4, 'vial', 'a vial full of a red liquid', true, false, false, true));
+    itemCollection.addItem(new Item(5, 'key', 'The key fits perfectly into the drawers in the desk!\nYou open each drawer to find them all empty....except for one. There\'s a small button hidden deep inside the drawer.\nYou press the button and a section of the western wall slides back to reveal a hidden entry into a bathroom!', true, false, true, false));
+    itemCollection.addItem(new Item(6, 'bandages', 'You use the bandages to wrap the cut on your hand.\nThe bleeding stops! You may just survive this house!', true, false, false, false));
+
+    // create untakeable items
+    itemCollection.addItem(new Item(1, 'desk', '', false, false, false, false));
+    itemCollection.addItem(new Item(1, 'bookcase', '', false, false, false, false));
+    itemCollection.addItem(new Item(5, 'lage desk', '', false, false, false, false));
+    itemCollection.addItem(new Item(5, 'portrait', '', false, false, false, false));
+    itemCollection.addItem(new Item(3, 'drawer', '', false, false, false, false));
+    itemCollection.addItem(new Item(4, 'table', '', false, false, false, false));
+
+    // Create player and display initial room description
+    let player = new Player();
+    player.observe();
+
+    let validCommands = {
+        take: ['get', 'grab', 'pickup'],
+        use: ['push', 'apply', 'put'],
+        exit: ['quit', 'bye']
+    }
```

Comment:
I don't believe that this command look up table is being used anywhere, If it's not you can remove it to cut down on some lines of code overall